Source/PeleLMEvolve.cpp

```diff
@@ -80,8 +87,34 @@ PeleLM::writePlotNow()
 
    if ( m_plot_int > 0 && (m_nstep % m_plot_int == 0) ) {
       write_now = true;
+   } else if (m_plot_per > 0.0) {
```

Comment:
Thanks for doing that !
I was initially planning on pulling it out of incflo with the option of either doing it approximately at the m_plot_per or enforcing exactly the period by shortening the dt. We can probably update that later though ...